ansible/trex_settings.yml

```diff
@@ -31,6 +31,8 @@ dut_isolated_cpus: 1,3,5,7,9,11,13,15
 dut_interface_1: enp68s0f0
 dut_interface_2: enp68s0f1
 
+dut_interface_1_pciid: "0000:44:00.0"
```

Comment:
I'll have to go back and change the dpdk pvp script as I was detecting this address on the fly but with your adding of this setting to the settings file I no longer need to do that.  I'll push up a commit later.

Reply:
yes, in pvp_ovsdpdk.yml, as the interface is still using kernel driver, you can get the pciid based on the name. but on pvp_dut.yml, the interface is already in dpdk, I can't get the pciid based on the interface name

Reply:
but for pvp_ovsdpdk.yml. we can either keep it as it is now or change the script a little. both are OK. and I can help to change it if needed. how do you think